### ISSUE TITLE
npm install download incompatible MongoClient version

### ISSUE THREAD

**Issue by davidnussio** [feathersjs-ecosystem/feathers-mongodb#102]
This is not a bug but now installing mongodb npm download the 3.0.0-rc0 version.
MongoClient.connect return client instead db.

The code generated with feathers-cli failed with the error:
*TypeError: db.collection is not a function*

- Fix with downgrade mongodb version to 2.2.x
- Fix follow the example below

```javascript
const MongoClient = require('mongodb').MongoClient;
const service = require('feathers-mongodb');

MongoClient.connect('mongodb://localhost:27017/feathers').then(client => {
  app.use('/messages', service({
    Model: client.db('feathers').collection('messages')
  }));
  app.use('/messages', service({ Model, id, events, paginate }));
});
```

**Comment by daffl** [comment]
If you could update the example that would be great. The latest generator already has been updated in https://github.com/feathersjs/generator-feathers/pull/318.

**Comment by daffl** [comment]
Closed via #103
